code/game/objects/items/food/snacks.dm

```diff
@@ -163,3 +163,13 @@
 	tastes = list("pure electricity" = 3, "fitness" = 2)
 	foodtypes = TOXIC
 	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/food/bluespacebar
+	name = "Bluespace Bar"
+	icon_state = "bluespacebar"
+	desc = "Probably paraphysical, definitely delicious. Orginally designed to help with running through walls."
+	trash_type = /obj/item/trash/bluespacebar
+	food_reagents = list(/datum/reagent/bluespace = 30, /datum/reagent/drug/methamphetamine = 10, /datum/reagent/consumable/nutriment = 5, /datum/reagent/consumable/sugar = 10)
```

Comment:
what da?

> buy 3 bluespace bars using roundstart assistant funds
> grind it down
> instant roundstart meth pills acquired

maybe have it not include meth

Reply:
I think the speedup given by the meth is important to the overall feel of the bar, but if comparatively easy meth is such a big issue I'll be happy to change it out for something else, like epinephrine.

Reply:
> buy 3 bluespace bars using roundstart assistant funds

theres only 1 stocked in the vendor, though theres perhaps a few vendors laying around, also as a contraband item the price is beyond assistant starting funds as far as im aware, or atleast 2+ would be

Reply:
if the meth is to avoid being stunned by the bluespace reagent, then the proper solution is to make a /datum/reagent/bluespace subtype that doesn't stun you

Reply:
> 
> 
> if the meth is to avoid being stunned by the bluespace reagent, then the proper solution is to make a /datum/reagent/bluespace subtype that doesn't stun you

the bluespace powder doesnt actually stun you, it does make you shake/jitter, but you dont get stunned

Reply:
Oh, i see. Still kinda weird